src/SWP/Bundle/BridgeBundle/Resources/config/doctrine-orm/Item.orm.yml

```diff
@@ -0,0 +1,54 @@
+SWP\Component\Bridge\Model\Item:
```

Comment:
There is no out of the box solution for this. We would have to create a custom mapping type. The existing Doctrine mapping types for array are ArrayType, which calls serialize and unserialize on the array, SimpleArrayType, which calls implode and explode, and JsonArrayType, which calls json_encode and json_decode. The Doctrine way of storing arrays of objects within an object is to store those objects in a separate table with a foreign key reference. The collection is only hydrated when the objects are required. Migration is supported. Serializing and deserializing an array is not an utterly trivial operation. So, I doubt that flattening an array of objects and storing it in one column is even optimal anyway.